### DISCUSSION TITLE
How to SSH from Vscode to VM VirtualBox with out error

### DISCUSSION THREAD

**Discussion by saclif** [microsoft/vscode-discussions#534] [Extension Development QnA] [answered]
Each time I try to ssh into a  Oracle VM VirtualBox Get the following error, and i do not know how to resolve it.  Can anyone help me?
![Capture](https://user-images.githubusercontent.com/128437519/226497086-69498259-3529-4a83-a2d4-90676814020e.JPG)

Error: Could not establish connection to "PC-PiOS": Could not resolve hostname.

**Comment by isidorn** [comment, marked as answer]
Please ask this on StackOverflow. 
Also these docs should help https://code.visualstudio.com/docs/remote/ssh

**Comment by saclif** [comment]
Isidor,

The links sent refers to Azure Portal for running a Virtual Machine and I am using Oracle VM VitualBox.

How can a person get some help getting VS-Code to run on VirtualBox?  Or must I use Azure Portal?

Thanks
ScottC
